src/contract/interfaces/IAdjudicationContract.ts

```
@@ -1,7 +1,19 @@
 import { Bytes, BigNumber, List } from '../../types/Codables'
-import { Property } from '../../ovm'
+import { ChallengeGame, Property } from '../../ovm'
 
 export interface IAdjudicationContract {
+  /**
+   * Gets instantiated challenge game by gameId
+   * @param gameId
+   */
+  getGame(gameId: Bytes): Promise<ChallengeGame>
```

what if a game doesn't exist for the given id? returns null?

---

can we throw exception if game is not found?

---

sure can.
I think it's good time to create custom error classes.

---

ok! I added about exception to comment.